test/test_testing.py

```diff
@@ -683,5 +687,99 @@ def test_calculate_2_shards_against_optimal_shards(self):
                 self.assertEqual(sorted_tests, sorted_shard_tests)
 
 
+skip_if_no_cuda = unittest.skipIf(not TEST_CUDA, "CUDA unavailable")
+
+
+class TestAsserts(TestCase):
+    def test_assert_tensors_equal_not_tensors(self):
+        a = torch.empty(())
+        b = np.empty(())
+
+        with self.assertRaises(AssertionError):
+            torch.testing.assert_tensors_equal(a, b)
+
+    @unittest.expectedFailure
+    def test_assert_tensors_equal_complex_support(self):
+        a = torch.ones(1, dtype=torch.float32)
+        b = torch.ones(1, dtype=torch.complex64)
+
+        torch.testing.assert_tensors_equal(a, b, check_dtype=False)
+
+    @unittest.expectedFailure
+    def test_assert_tensors_equal_sparse_support(self):
+        a = torch.empty(())
+        b = torch.sparse_coo_tensor(size=())
+
+        torch.testing.assert_tensors_equal(a, b)
+
+    @unittest.expectedFailure
+    def test_assert_tensors_equal_quantized_support(self):
+        a = torch.ones(1, dtype=torch.int32)
+        b = torch.ones(1, dtype=torch.qint32)
+
+        torch.testing.assert_tensors_equal(a, b)
```

Comment:
This fails, but in the attempted creation of a quantized tensor rather than our assert. Is it possible to directly create a quantized tensor without going through all the steps? We simply need a tensor for that `t.is_quantized is True`,